index/scorch/persister.go

```diff
@@ -497,14 +497,14 @@ func (s *Scorch) persistSnapshotDirect(snapshot *IndexSnapshot) (err error) {
 
 		select {
 		case <-s.closeCh:
-			err = ErrClosed
+			err = zap.ErrClosed
```

Comment:
thinking this might be cleaner if it returned bleve/index/scorch/segment.ErrClosed?

and, might as well do the one-liner of returning the error?